.github/workflows/ci.yml

```diff
@@ -110,18 +110,42 @@ jobs:
     needs: [test]
     if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out Git repository
+        uses: actions/checkout@v1
```

Comment:
why downgrading to checkout@v1?

Reply:
Other actions from the `actions` namespace also seem to be outdated. I'd recommend updating them all.